WTG.Analyzers.Test/TestData/CodeContractsAnalyzer/Requires/Unknown/Source.cs

```diff
@@ -0,0 +1,37 @@
+using System.Diagnostics.Contracts;
+
+public class Bob : IBob
+{
+	public void Method1(int value)
+	{
+		Contract.Requires(value > 3);
+
+		value.GetType();
+	}
+
+	protected void Method2(int value)
+	{
+		Contract.Requires(value > 3, "Message");
+
+		value.GetType();
+	}
+
+	void IBob.Method(int value)
+	{
+		Contract.Requires(value > 3);
```

Comment:
Isn't this technically invalid? 🙃

Reply:
Yeah, but you only notice if the rewriter is running and "CONTRACTS_FULL" is defined :)